src/transformers/utils/doc.py

```diff
@@ -560,17 +560,26 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
     >>> import tensorflow as tf
 
     >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
-    >>> model = {model_class}.from_pretrained("{checkpoint}")
+    >>> model = {model_class}.from_pretrained("{checkpoint}", from_pt=True)
```

Comment:
TODO: upload TF checkpoint and remove from_pt

Reply:
yeah we can't do this here :-) Think if you upload the models under your name or you find other well-working TF ones that be perfect